src/Message/Organization/EnableSecurityScan.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buddy\Repman\Message\Organization;
+
+final class EnableSecurityScan
```

Comment:
hmm, this command can enable and disable security scan, so maybe we can pick better name, for example: `ChangeSecurityScan` or `UpdateSecurityScanOption` ... wdyt?

Reply:
Yeah, I was a bit unsure how to properly name it. I felt it would be good to name the message like the flag I introduced, but since the message will toggle the flag it sounds a bit wired.

We can surely go with `ChangeSecurityScan` - I'll think about it, maybe I can come up with a better name. Will push later today...

Reply:
Went with `ChangeSecurityScanConfiguration`.